demo/index.html

```diff
@@ -46,7 +46,7 @@ <h2>Must be uploaded to the site for make cookies work (not working as local fil
 					<td role="cell">90</td>
 					<td role="cell">5400</td>
 					<td role="cell">176</td>
-					<td class="price" role="cell">2087.79 Р</td>
+					<td class="price" role="cell">208.79 $</td>
```

Comment:
@BeastJsWeb Please try to think how it is possible to dynamically fix currency here?

Reply:
Ok, I will see